src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -166,7 +166,7 @@ class Proxied<T extends ProxyTarget> {
       },
       apply: (target, thisArg, argArray) => {
         if (this.appliedValue) {
-          return this.appliedValue.apply(thisArg, argArray);
+          return this.appliedValue.apply(thisArg);
```

Comment:
Is this intentional? This will remove all the positional arguments

Reply:
Yes, but only because appliedValue is defined as `() => any` - it doesn't (in theory) have any arguments. The typing got more strict around this. Let me see if I can just make it accept anything, that might be the safer route (but unsure how much work it will be)

Reply:
Ah, yeah, okay. I fixed it, I think. Thanks for flagging.